processing/data_collection/gazette/spiders/mg_belo_horizonte.py

```diff
@@ -0,0 +1,49 @@
+from datetime import date, datetime, timedelta
+from functools import partial
+
+from scrapy import Request, Spider
+
+from gazette.items import Gazette
+
+
+class MgBeloHorizonteSpider(Spider):
+    MUNICIPALITY_ID = 3106200
+    name = "mg_belo_horizonte"
+    allowed_domains = ["portal6.pbh.gov.br"]
+    documents_url = (
+        "http://portal6.pbh.gov.br/dom/iniciaEdicao.do"
+        "?method=DomDia&dia={}&comboAno={}"
+    )
+    first_edition = date(2002, 3, 21)
+
+    def start_requests(self):
+        current_date = date.today()
+        while current_date >= self.first_edition:
+            formatted_date = current_date.strftime("%d/%m/%Y")
+            yield Request(
+                self.documents_url.format(formatted_date, current_date.year),
+                callback=partial(self.parse_page_link, current_date),
```

Comment:
Usually we use `Request.meta` dictionary to send information between requests (https://doc.scrapy.org/en/latest/topics/request-response.html#scrapy.http.Request.meta).